docs/hw-platforms.rst

```diff
@@ -47,13 +47,13 @@ The following XMC microcontroller boards are supported by XMC for Arduino:
 KIT_XMC11_BOOT_001
 ------------------
 
-* Name in Arduino IDE: ``XMC1100 Boot Kit``
+* Legacy name: ``XMC1100 Boot Kit``
```

Comment:
Why do we keep the legacy name? 
The idea of renaming is to have unify and simplify to one name.
If the user get´s lost, maybe a look up table with legacy name and new name would be convenient? But then keep it in a section decouple from the main doc, so that we have a clean single naming approach across code and docs.